ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -135,27 +106,27 @@ protocolInfoShelley genesis protVer mbCredentials =
     consensusConfig :: ConsensusConfig (BlockProtocol (ShelleyBlock c))
     consensusConfig = TPraosConfig {
         tpraosParams
+      , tpraosMaxMajorPV = maxMajorPV
       , tpraosEpochInfo = epochInfo
       }
 
     ledgerConfig :: LedgerConfig (ShelleyBlock c)
-    ledgerConfig = mkShelleyLedgerConfig genesis epochInfo
+    ledgerConfig = mkShelleyLedgerConfig genesis epochInfo maxMajorPV
 
     -- TODO: This must instead be derived from the hard fork history.
     -- <https://github.com/input-output-hk/ouroboros-network/issues/1205>
     epochInfo :: EpochInfo Identity
-    epochInfo = fixedSizeEpochInfo $ sgEpochLength genesis
+    epochInfo = fixedSizeEpochInfo $ SL.sgEpochLength genesis
 
     tpraosParams :: TPraosParams
     tpraosParams = TPraosParams {
-        tpraosSlotsPerKESPeriod = sgSlotsPerKESPeriod genesis
-      , tpraosLeaderF           = sgActiveSlotCoeff   genesis
-      , tpraosSecurityParam     = sgSecurityParam     genesis
-      , tpraosMaxKESEvo         = sgMaxKESEvolutions  genesis
-      , tpraosQuorum            = sgUpdateQuorum      genesis
-      , tpraosMaxMajorPV        = sgMaxMajorPV        genesis
-      , tpraosMaxLovelaceSupply = sgMaxLovelaceSupply genesis
-      , tpraosNetworkId         = sgNetworkId         genesis
+        tpraosSlotsPerKESPeriod = SL.sgSlotsPerKESPeriod genesis
+      , tpraosLeaderF           = SL.sgActiveSlotCoeff   genesis
+      , tpraosSecurityParam     = SecurityParam $ SL.sgSecurityParam genesis
+      , tpraosMaxKESEvo         = SL.sgMaxKESEvolutions  genesis
+      , tpraosQuorum            = SL.sgUpdateQuorum      genesis
+      , tpraosMaxLovelaceSupply = SL.sgMaxLovelaceSupply genesis
+      , tpraosNetworkId         = SL.sgNetworkId         genesis
```

Comment:
Realign on `genesis`. Better yet, add:
```haskell
securityParam :: SecurityParam
securityParam = SecurityParam $ SL.sgSecurityParam genesis
```
(at the same level as `tpraosParams` is fine.)